### DISCUSSION TITLE
[BUG] 2 out of rest position are bugged

### DISCUSSION THREAD

**Discussion by CaptLemur** [ErinaSugino/Starbound-SBR-Mod#127] [Trouble Shooting]
For some reasons i only have to positions available (standing and doggy) but rest makes NPC's disappear in thin air leaving my char in position.
It donsnt crashes or anything, but after leaving a scene my copy of my self is made still in position

For now I cant use:
missionary
face sitting
cow girl
blow job
butterfly
reverse cowgirl

**Comment by ErinaSugino** [comment]
Can't help you without a log file.

**Reply by CaptLemur** [reply]
> Can't help you without a log file.

I hope they will help you with that.
I will send 5 of them from my previous game (I suppose 2 of them might not having it cus of normal gameplay)
Also if you want me to do one more session with only that bug I have no problem doing it

[Logs Starbound - sxb position bug.zip](https://github.com/user-attachments/files/16599650/Logs.Starbound.-.sxb.position.bug.zip)

**Reply by ErinaSugino** [reply]
What I read from this is, that you are using the new Neki SxB 3.0 version which has the animated ears.
However at the same time, you are using an outdated version of the GIC Sexbound patch, which overrides the animation files with an old version where the ears layer does not yet exists for no reason. I have contacted the creator of the GIC patch in the past and it should have been fixed, so just updating/redownloading the GIC patch should fix your issue.

**Reply by CaptLemur** [reply]
> What I read from this is, that you are using the new Neki SxB 3.0 version which has the animated ears. However at the same time, you are using an outdated version of the GIC Sexbound patch, which overrides the animation files with an old version where the ears layer does not yet exists for no reason. I have contacted the creator of the GIC patch in the past and it should have been fixed, so just updating/redownloading the GIC patch should fix your issue.

soo its GIC patch at fault then?
didn't really thought about it tbh.

Also its fine with neki sxb tho?
Wont do any hibbi-habby?

Also if I remember last gic patch was in may 30 (like 3 months ago)
Soo still that would cause an issue that last update of this mod then.
Cus I downloaded that patch from that period time, unless creator of mod has plans to update it.
If anything you know about other gic patch mod then I would be happy to get it.

**Reply by ErinaSugino** [reply]
The last update is indeed from the 30th of May, which is the hotfix that should have fixed this issue. Have you downloaded the mod since then or do you have a version from before?

NekiSxB will always be fine. It's literally the single most compatible race with Sexbound, and I'll personally make sure it stays that way.

**Reply by CaptLemur** [reply]
> The last update is indeed from the 30th of May, which is the hotfix that should have fixed this issue. Have you downloaded the mod since then or do you have a version from before?
> 
> NekiSxB will always be fine. It's literally the single most compatible race with Sexbound, and I'll personally make sure it stays that way.

I downloaded the latest from may 30
seems like it hasn't been fixed... unless GIC mod was updated 
As I write this, workshop server of steam are down its hard for me to check it now 

Edit:
GIC was updated in may 2, soo its seems like the gic patch isn't fully working guessing by this

**Reply by ErinaSugino** [reply]
Well, if that's not the solution I'm afraid you have no other choice than to back up your savefile, remove all non-Steam mods except Sexbound itself and the NekiSxB mod, and start re-adding them one by one until the culprit appears.
